packages/api/tests/lambdas/test-bulk-granule-delete.js

```diff
@@ -127,19 +127,7 @@ test('bulkGranuleDelete does not fail on published granules if payload.forceRemo
       ],
       forceRemoveFromCmr: true,
     },
-    async ({ _, pgGranuleRecord }) => {
-      const dynamoGranule = await granuleModel.get(
-        { granuleId: pgGranuleRecord.granule_id }
-      );
-
-      return {
-        dynamoGranule: { ...dynamoGranule, published: false },
-        pgGranule: {
-          ...pgGranuleRecord,
-          published: false,
-        },
-      };
-    }
+    () => true
```

Comment:
Could we add a comment here?

Reply:
Yes, but I think making it into a descriptive variable probably works as well/better - how's this: a4e5d9a9c5